addons/l10n_id_efaktur_coretax/models/uom_code.py

```diff
@@ -8,3 +8,9 @@ class EfakturUomCode(models.Model):
 
     code = fields.Char()
     name = fields.Char()
+
+    def name_get(self):
+        result = []
+        for record in self:
+            result.append((record.id, f"{record.code} - {record.name}"))
```

Comment:
I would do "name (code)", do you think it would be clearer?

Reply:
That sounds good too!